src/input.js

```diff
@@ -0,0 +1,84 @@
+import Car from './car.js';
+import startRacing from './game.js';
+import { ERROR_INPUT } from './define-msg.js';
+
+const carObjectArray = [];
+let racingCount = 0;
+
+function checkEmptyInput(carNameInput) {
+  return carNameInput.trim() === '';
+}
+
+function checkDoubleElements(carNameArray) {
+  return new Set(carNameArray).length === carNameArray.length;
+}
+
+function checkValidCarNameLength(carNameArray) {
+  for (let index = 0; index < carNameArray.length; index += 1) {
+    if (carNameArray[index].length > 5) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
```suggestion
 return carNameArray.filter(x=>x.length > 5) > 0;
```
filter를 사용해서 조금 더 가독성있고 짧은 코드를 짤 수 있을 것 같아서 리뷰했습니다!

Reply:
넘 좋습니다. 배열 메서드를 많이 찾아봤는데 역시 이렇게 직접 배우는게 훨씬 효과가 있는 것 같아요. 고마워요.